### PR TITLE
.NET 5.0

### PR DESCRIPTION
**Pull request by NN---**
@vtbassmatt

### CODE REVIEW

**Comment by NN---** [comment]
@vtbassmatt Not sure what is the problem `license/cla` check.

**Review by derrickstolee — COMMENTED** [review]
I just pointed out one small bit. We will want to wait for @mjcheetham who knows a bit about why the Windows version is still on .NET Framework (or is it Standard?).

File: .github/ISSUE_TEMPLATE/auth-problem.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ assignees: ''
 From a terminal, run `git-credential-manager-core version` and paste the output.
 
 <!-- Ex:
-Git Credential Manager version 2.0.8-beta+e1f8492d04 (macOS, .NET Core 4.6.27129.04)
+Git Credential Manager version 2.0.8-beta+e1f8492d04 (macOS, .NET 4.6.27129.04)
```
[the remark this answers is not in the document]
Comment (NN---) [reply .github/ISSUE_TEMPLATE/auth-problem.md:14]: .NET Core4.6 doesn’t exist . Is this a typo and should be .NET Framework?

[the remark this answers is not in the document]
Comment (NN---) [reply .github/ISSUE_TEMPLATE/auth-problem.md]: Reverted

**Comment by NN---** [comment]
> I just pointed out one small bit. We will want to wait for @mjcheetham who knows a bit about why the Windows version is still on .NET Framework (or is it Standard?).

https://github.com/microsoft/Git-Credential-Manager-Core/issues/113

**Comment by normanlorrain** [comment]
The URL in README.md is incorrect (.NET should point to https://dotnet.microsoft.com/):
Line 9 should be
` 
[Git Credential Manager Core](https://github.com/microsoft/Git-Credential-Manager-Core) (GCM Core) is a secure Git credential helper built on [.NET](https://dotnet.microsoft.com/) that runs on Windows and macOS. Linux support is in an early preview.
`

**Comment by NN---** [comment]
> 
> A URL in README.md is incorrect (.NET should point to [dotnet.microsoft.com](https://dotnet.microsoft.com/)):
> Line 9 should be

@normalian I can change, but probably it is better to be done as a separate PR.

**Comment by NN---** [comment]
Fixed here and as a separate PR #293

**Review by mjcheetham — APPROVED** [review]
This is and looks great thanks @NN--- ! 👍

**Pull request merged**